README.md

```diff
@@ -109,8 +109,7 @@ bundle exec rake m2c_exist_single_root[fixture_sr1]
 ```ruby
 bundle exec rake m2c_exist_single_root[fixture_sr1,profile]
 ```
-this will generate a log at, for example, ```log/profiler_flat_check_existence_for_dir2017-12-11T14:34:06-flat.txt
-```
+this will generate a log at, for example, ```log/profiler_flat_check_existence_for_dir2017-12-11T14:34:06.txt```
```

Comment:
I believe, besides fixing the README, you will need to fix the actual code generating this file name

Reply:
on that note, i think i'd prefer that we get rid of the first `_flat` and leave the last `-flat`, since the latter is put there by the profiler code (which seems like the more logical bit of code to note the type of profile data in the logfile name, esp since its also the code responsible for constructing the timestamp and file extension parts of the file name).

Reply:
i removed the `_flat`  and left the `-flat` in.